### PR TITLE
Add data storage

### PR DESCRIPTION
**Pull request by mattandrews**
Work in progress

### CODE REVIEW

File: models/index.js [hunk 1]
```diff
@@ -38,14 +38,15 @@ sequelize
...
-db.Staff = sequelize.import('../models/staff.js');
+db.Users = sequelize.import('./user.js');
+db.Staff = sequelize.import('./staff.js');
 db.Feedback = sequelize.import('./feedback');
+db.Application = sequelize.import('./application');
 
 db.SurveyAnswer = sequelize.import('./survey');
 
-db.Order = sequelize.import('../models/materials/order.js');
-db.OrderItem = sequelize.import('../models/materials/orderItem.js');
```
Comment (davidrapson) [inline models/index.js]: The `.js` extension can also go fyi.

File: controllers/apply-next/views/startpage.njk [hunk 2]
```diff
@@ -12,10 +13,35 @@
                 suffix = copy.description | safe
             ) }}
 
-            <h2>Introduction</h2>
-            <p>Form intro goes here</p>
+            {% if applications %}
```
Comment (davidrapson) [inline controllers/apply-next/views/startpage.njk]: We'll probably need to distinguish between the "dashboard" page, where your applications live, and the "startpage"  which summaries the form and includes basic guidance. This is fine for now though.

Comment (mattandrews) [reply controllers/apply-next/views/startpage.njk]: Yeah – I figured at this point we only want to show you applications for *this* form, but yeah, we'll tweak this later.

File: controllers/apply-next/form-router.js [hunk 3]
```diff
@@ -196,29 +226,92 @@ function initFormRouter(formModel) {
         next();
     });
 
+    // Require login before using everything else after this
+    router.use((req, res, next) => {
```
Comment (davidrapson) [inline controllers/apply-next/form-router.js]: This should probably be moved up above (or in to?) the previous `req.use`. Currently it looks like we're doing a bunch of work in the previous middleware that might not apply if the user isn't logged in yet.

File: services/applications.js [hunk 4]
```diff
@@ -1,17 +1,75 @@
 'use strict';
-const request = require('request-promise-native');
-const { APPLICATIONS_SERVICE_ENDPOINT } = require('../modules/secrets');
```
Comment (davidrapson) [inline services/applications.js]: Can probably remove `APPLICATIONS_SERVICE_ENDPOINT` from `modules/secrets` too

**Pull request merged**